packages/jazz-tools/src/implementation/zodSchema/runtimeConverters/zodSchemaToCoSchema.ts

```diff
@@ -61,19 +61,6 @@ export function tryZodSchemaToCoSchema<S extends z.core.$ZodType>(
         }
       };
 
-      if ("migration" in schema) {
-        const migration = schema.migration;
-        if (typeof migration !== "function") {
-          throw new Error("migration must be a function");
-        }
-        (coSchema.prototype as Account).migrate = async function (
-          this,
-          creationProps,
-        ) {
-          await migration(this, creationProps);
-        };
-      }
```

Comment:
Changed the approach to patch the schema prototype directly in the definer functions.